app/views/shared/_tea_time_row.html.erb

```diff
@@ -70,8 +70,20 @@
             </button>
           <% end %>
         <% end %>
+      <% elsif (no_signin_button ||= false) # No signin button; don't show signin CTA text %>
```

Comment:
On the `/tea_times` page with the new flow, we only want to display "Count Me In" and "Join Waitlist" buttons. We don't want to display "Sign in" buttons, so we have a special branch for the `/tea_times` page.